packages/smooth_app/lib/pages/scan/scan_page.dart

```diff
@@ -66,7 +67,9 @@ class _ScanPageState extends State<ScanPage> {
                     case DevicePermissionStatus.checking:
                       return EMPTY_WIDGET;
                     case DevicePermissionStatus.granted:
-                      return CameraScannerPage(scannerType);
+                      // TODO(m123): change
+                      return const CameraScannerPage(
+                          SmoothBarcodeScannerType.mockup);
```

Comment:
This I don't get.

Reply:
You probably will not, as it makes no sense. It is just a lazy way of me saying. Okay we'll not use this for the time being, but I will not remove it as we'll need it for the coming release. Just a commit with very bad code to be changed in a coming one